examples/exec-command.js

```diff
@@ -0,0 +1,42 @@
+
+import { sleep } from 'k6';
+import { Kubernetes } from 'k6/x/kubernetes';
+
+export default function () {
+  const kubernetes = new Kubernetes({
+  })
+  const namespace = "default"
+  const podName = "new-pod"
+  const image = "busybox"
+  const command = ["sh",  "-c", "sleep 300"]
+
+  kubernetes.pods.create({
+    namespace: namespace,
+    name: podName,
+    image: image,
+    command: command
+  })
+  sleep(3)
+
+  const newPod = kubernetes.pods.list(namespace).find(function(pod) { return pod.name == podName}) 
+  if (!newPod) {
+    throw podName + " pod was not created"
+  }
+
+  const container = newPod.spec.containers[0].name
+  const result = kubernetes.pods.exec({
+    namespace: namespace,
+    pod: podName,
+    container: container.name,
```

Comment:
@pablochacin 

Reviewing the example from a user perspective, it would be ideal to skip passing the container name.

What makes me think if we could pass the pod directly , so either the `podName` or the `pod`.


```javascript
kubernetes.pods.exec({
  pod: pod,
  command:[]
  stadin: []
});
```

Reply:
Pods can have multiple containers, so it is important that the API allows specifying the target container.

For example,  when using ephemeral containers to add chaos behavior to a running pod, it is necessary to be able to execute commands in the ephemeral container.

What could be improved is to mimic the behavior of the kubectl command and chose a default container if none is specified.

Reply:
> What could be improved is to mimic the behavior of the kubectl command and chose a default container if none is specified.

Personally, I never liked this behavior :( 

So, I would keep it as it is right now.